cmd/torrent-verify/main.go

```diff
@@ -16,16 +16,40 @@ import (
 )
 
 var (
-	filePath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
-	dirPath  = flag.String("path", "/torrent/data", "path of the torrent data")
+	torrentPath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
+	dataPath    = flag.String("path", "/torrent/data", "path of the torrent data")
+	summary     = flag.Bool("summary", false, "display summary at the end")
 )
 
-func init() {
-	flag.Parse()
+func verifySummary(sMap map[bool][]int) {
+	fmt.Println("----------------")
+	fmt.Println(" TORRENT-VERIFY ")
+	fmt.Println("----------------")
+	fmt.Printf("Number of correct pieces: %d\n", len(sMap[true]))
+	fmt.Printf("Number of wrong pieces: %d\n", len(sMap[false]))
```

Comment:
I kinda like the original idea of having a visibility of which piece is incorrect.  I'm happy to implement the progress bit, but like I said we will lose the per-piece visibility, which does not necessarily have to be a bad thing ?